__tests__/gui/toggling_indicators.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Facility Modal
+
+  Scenario: Verify the print view is correct
+    Given I am on the SANEF Homepage
+      Then I wait until map is ready
+    And I expand Data Mapper
+      Then Data Mapper should be displayed
+    And I click on Demographics in Data Mapper
+      Then I select an indicator
+    And I select another indicator
+      Then I check if everything is zero
```

Comment:
Wouldn't it be better to call this  `Then I check that everything is not zero`?

Reply:
updated the step description